File: Selenium/Project/Activity2.java
package HRM;
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.testng.annotations.AfterTest;
import org.testng.annotations.BeforeTest;
import org.testng.annotations.Test;

public class Activity2 {

        WebDriver driver;

        @BeforeTest
        public void setUp() {
            WebDriverManager.chromedriver().setup();
            //Create a new instance of the Firefox driver
            driver = new ChromeDriver();
            // Open the URL
            driver.get("http://alchemy.hguy.co/orangehrm");
        }

        @Test
        public void getHeaderImageURL() {
            // Get the URL of the header image using different selectors
            //WebElement headerImage = driver.findElement(By.cssSelector("img[src*='header']")); // Using CSS Selector
            // or
            WebElement headerImage = driver.findElement(By.xpath("//*[@id=\"divLogo\"]/img")); // Using XPath

            String headerImageURL = headerImage.getAttribute("src");

            // Print the URL to the console
            System.out.println("Header Image URL: " + headerImageURL);
        }

        @AfterTest
        public void tearDown() {
            driver.quit();
        }
    }

File: Selenium/Project/Activity2.py
from selenium import webdriver
from selenium.webdriver.chrome.service import Service as ChromeService
from webdriver_manager.chrome import ChromeDriverManager
from selenium.webdriver.common.by import By
driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()))
# Navigate to the HRM system login page
url = "http://alchemy.hguy.co/orangehrm"
driver.get(url)

# Find the header image element by its XPath (you may need to inspect the page to get the correct XPath)
header_image = driver.find_element(By.XPATH,"//*[@id='divLogo']/img")

header_image_url=header_image.get_property("src")
# Print the URL to the console
print("URL of the header image:", header_image_url)

# Close the browser
driver.quit()
